### PR TITLE
FCREPO-1871. Update fcrepo-camel-toolbox client code imports.

### PR DESCRIPTION
**Pull request by mohideen**
https://jira.duraspace.org/browse/FCREPO-1871

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -41,7 +41,8 @@
     <spring.version>4.1.5.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-audit.version>4.4.0</fcrepo-audit.version>
-    <fcrepo-camel.version>4.4.0</fcrepo-camel.version>
+    <fcrepo-camel.version>4.4.1-SNAPSHOT</fcrepo-camel.version>
```
Comment (mohideen) [inline pom.xml]: Note: The camel dependency version is changed to a SNAPSHOT version here. If this is not desired, we can wait until fcrepo-camel 4.4.1 is released. And, update and merge the PR after the release.

File: fcrepo-indexing-solr/pom.xml [hunk 2]
```diff
@@ -46,6 +46,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.fcrepo.client</groupId>
+      <artifactId>fcrepo-java-client</artifactId>
+    </dependency>
```
Comment (acoburn) [inline fcrepo-indexing-solr/pom.xml]: scope: test

File: fcrepo-indexing-triplestore/pom.xml [hunk 3]
```diff
@@ -41,6 +41,10 @@
       <artifactId>fcrepo-camel</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.fcrepo.client</groupId>
+      <artifactId>fcrepo-java-client</artifactId>
+    </dependency>
```
Comment (acoburn) [inline fcrepo-indexing-triplestore/pom.xml]: scope: test

File: fcrepo-serialization/pom.xml [hunk 4]
```diff
@@ -36,6 +36,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.fcrepo.client</groupId>
+      <artifactId>fcrepo-java-client</artifactId>
```
Comment (acoburn) [inline fcrepo-serialization/pom.xml]: scope: test

File: fcrepo-fixity/pom.xml [hunk 5]
```diff
@@ -36,6 +36,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.fcrepo.client</groupId>
+      <artifactId>fcrepo-java-client</artifactId>
```
Comment (acoburn) [inline fcrepo-fixity/pom.xml]: use scope: test

File: pom.xml (hunk 1 above)
Comment (acoburn) [inline pom.xml]: I would highly recommend keeping this in line with non-SNAPSHOT releases -- it means we can version the projects independently, which is highly desirable. If you would like to have a new `fcrepo-camel` release (with the new `fcrepo-java-client`), I can do that early next week.

**Comment by acoburn** [comment]
This should wait until there is a new release of `fcrepo-camel` (containing the updated `fcrepo-java-client`)

**Comment by mohideen** [comment]
I have updated dependency scope to test for modules that uses the fcrepo-java-client for test only.

**Comment by acoburn** [comment]
merged via e13924eb9427e8590c92754d91be90cc98e0b096

**Pull request closed without merging**